clusterctl/clusterdeployer/clusterclient.go

```diff
@@ -43,17 +44,18 @@ const (
 )
 
 type clusterClient struct {
-	clientSet      clientset.Interface
-	kubeconfigFile string
-	closeFn        func() error
+	clientSet       clientset.Interface
+	kubeconfigFile  string
+	configOverrides tcmd.ConfigOverrides
+	closeFn         func() error
 }
 
 func NewClusterClient(kubeconfig string) (*clusterClient, error) {
```

Comment:
This should just be `New(...)` so that people who import the package call `clusterclient.New(...)` instead of `clusterclient.NewClusterClient(...)` which is redundantly long (for go style).

Reply:
Moving it in another PR would be great. Want to open an issue and label it as help-wanted? That would be a good thing for someone who wants to start sending code to take a crack at.

Reply:
Created an issue here: https://github.com/kubernetes-sigs/cluster-api/issues/398